easybuild/easyconfigs/n/numba/numba-0.37.0-foss-2018a-Python-2.7.14.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = 'Bundle'
```

Comment:
@FokkeDijkstra Do you mind updating this to `PythonBundle`, which allows some cleanup below

Reply:
No problem. I've done this.